### PR TITLE
[CI] Migration CCI->GHA

### PR DESCRIPTION
**Pull request by vmoens**

### CODE REVIEW

File: .github/workflows/test-macos.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+jobs:
+  tests-intel:
+    strategy:
+      matrix:
+        python_version: ["3.8", "3.11"]
+      fail-fast: false
+    uses: pytorch/test-infra/.github/workflows/macos_job.yml@main
+    with:
+      repository: pytorch/rl
```
Comment (DanilBaibak) [inline .github/workflows/test-macos.yml:29]: Should the repository be pytorch/tensordict?

File: .github/workflows/test-macos.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+  tests-silicon:
+    strategy:
+      matrix:
+        python_version: ["3.8"]
+      fail-fast: false
+    uses: pytorch/test-infra/.github/workflows/macos_job.yml@main
+    with:
+      runner: macos-m1-12
+      repository: pytorch/rl
```
Comment (DanilBaibak) [inline .github/workflows/test-macos.yml:55]: Should the repository be pytorch/tensordict?

**Review by DanilBaibak — APPROVED** [review]
I have one tiny question. Otherwise LGTM!

**Comment by vmoens** [comment]
> I have one tiny question. Otherwise LGTM!

Thanks i made the edits!

**Pull request merged**